media-sound/picard/picard-1.4.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+DISTUTILS_SINGLE_IMPL=1
+DISABLE_AUTOFORMATTING=true
+inherit eutils distutils-r1 readme.gentoo-r1
+
+DESCRIPTION="A cross-platform music tagger"
+HOMEPAGE="https://picard.musicbrainz.org"
+SRC_URI="https://github.com/metabrainz/${PN}/archive/release-${PV}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="+acoustid +cdda nls"
+
+DEPEND="dev-python/PyQt4[X,${PYTHON_USEDEP}]
+	dev-qt/qtgui:4[accessibility]
+	media-libs/mutagen
+	acoustid? ( >=media-libs/chromaprint-1.0[tools] )
+	cdda? ( dev-python/python-discid )"
+RDEPEND="${DEPEND}"
+
+RESTRICT="test" # doesn't work with ebuilds
+S=${WORKDIR}/${PN}-release-${PV}
+DOCS="AUTHORS.txt NEWS.txt"
+
+DOC_CONTENTS="If you are upgrading Picard and it does not start,
+try removing Picard's settings:
+    rm ~/.config/MusicBrainz/Picard.conf
+
+You should set the environment variable BROWSER to something like
+    firefox '%s' &
+to let python know which browser to use."
```

Comment:
metabrainz/picard@e2872b8275fd26c315956a74eaae64c43e26299b added support for `xdg-open`, so it should work on any desktop environment, as long as `x11-misc/xdg-utils`is installed.

Even without `xdg-open`, the Python[ `webbrowser`](https://docs.python.org/2/library/webbrowser.html) module has a built-in list of browsers that it will try to open.

My main objection is to the "should". It seems far too strong for something that only affects a very small number of users who probably already know about the environment variable anyway.